lms/djangoapps/courseware/tabs.py

```diff
@@ -299,9 +299,12 @@ def get_course_tab_list(request, course):
         if must_complete_ee:
             # Hide all of the tabs except for 'Courseware'
             # Rename 'Courseware' tab to 'Entrance Exam'
-            if tab.type is not 'courseware':
+            if tab.type != 'courseware':
                 continue
             tab.name = _("Entrance Exam")
+        if tab.type == 'static_tab' and tab.instructors_only and \
+                not bool(user and has_access(user, 'staff', course, course.id)):
```

Comment:
Our preferred way of accessing this information is CourseStaffRole(course_key).has_user(user).
Hopefully that will also allow you to get it on a single line, as we try to avoid "\". :)